internal/stack/_static/elasticsearch.yml.tmpl

```diff
@@ -21,3 +21,10 @@ script.context.ingest.cache_max_size: 2000
 script.context.processor_conditional.cache_max_size: 2000
 script.context.template.cache_max_size: 2000
 {{ end }}
+
+{{ $apm_server_enabled := fact "apm_server_enabled" }}
+{{ if eq $apm_server_enabled "true" }}
+tracing.apm.enabled: true
```

Comment:
I like the idea of always turning apm instrumentation on when enabling apm-server - how about we rename the config to `apm_enabled` to reflect that?

Reply:
cc @ChrsMark who wanted to use this.

Reply:
My use-case is shipping OTel data to that local APM Server. So once this is covered no strong opinion on how this enablement config will be called.